docs/source/install/k8s_ha.rst

```diff
@@ -46,16 +46,17 @@ makes installing the complex StackStorm infrastructure as easy as:
   # Add Helm StackStorm repository
   helm repo add stackstorm https://helm.stackstorm.com/
 
-  helm install stackstorm/stackstorm-ha
+  # Install StackStorm HA with an automatically-generated release name in the "stackstorm" namespace
+  # Replace "--generate-name" with a release name if you would like to name the deployment
+  # Omit the "--namespace stackstorm" flag if you would like to deploy to the "default" namespace
+  # Add "--create-namespace" if the namespace that you are specifying needs to be created
+  helm install --generate-name stackstorm/stackstorm-ha
 
-Once the deployment is finished, it will show you the first steps to get started working with the new cluster via WebUI
-or ``st2`` CLI client:
+After the installation completes, it will display a message similar to the following:
```

Comment:
Looks like the previous message was more detailed/specific:
```suggestion
Once the deployment is finished, it will show you the first steps to get started with the new cluster via WebUI
or ``st2`` CLI client:
```